### PR TITLE
Linter_Bears_Advanced: Fix typo

### PR DESCRIPTION
**Pull request by ventris**
Change the variable name value=2 to value2
Closes https://github.com/coala/documentation/issues/140

### CODE REVIEW

**Comment by ventris** [comment]
Reopened https://github.com/coala/documentation/pull/142 as I was not able to resync it.

File: Developers/Linter_Bears_Advanced.rst [hunk 1]
```diff
@@ -34,7 +34,7 @@ The path of the temporary configuration file can be accessed inside
         @staticmethod
```
Comment (adtac) [inline Developers/Linter_Bears_Advanced.rst]: As a general rule of thumb, we avoid words like "Fix" in our commit message. They're often filler words that may hide the actual commit changes. I'd suggest changing the commit head to `Linter_Bears_Advanced: value=2 -> value2` as that's much more meaningful IMO.

**Comment by ventris** [comment]
Welp, there is nothing about that on the wiki and some other people left comments in the last PR on the same issue that I should just specifically "fix typo". So I am rather confused on what I should or should not do.

**Comment by ventris** [comment]
I have changed the shortlog to reflect your request of change.

**Comment by adtac** [comment]
Hey @ventris this is almost good to go. Just one change:

we use some bug prediction in coala which relies on the `Fixes` keyword in commit messages. To get good results from that we need to use `Closes` for normal issues instead of `Fixes` which should only be used for real bugs. (See also [the commit message docs](http://coala.io/commit).)

To change your message you just use `git commit --amend` and then `git push --force` the new commit to replace the old one.

If you're just looking to fix an issue very quickly and not interested in contributing to coala long term, we can fix up the message for you - just tell us :).

Cheers!

**Comment by adtac** [comment]
ack https://github.com/coala/documentation/pull/158/commits/f54ec1d57ed8241f577295177887dc5c04719e05

**Comment by adtac** [comment]
@rultor merge

**Pull request merged**